### PR TITLE
docs: update service-worker-getting-started page

### PR DESCRIPTION
**Pull request by LiadIdan**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[x] Bugfix
```

## What is the current behavior?

Issue Number: #25783

## What is the new behavior?
Preventing cache collision with the service worker by adding the `-c-1` parameter when starting `http-server`

## Does this PR introduce a breaking change?
```
[x] No
```

### CODE REVIEW

**Comment by LiadIdan** [comment]
I signed it!

File: aio/content/guide/service-worker-getting-started.md [hunk 1]
```diff
@@ -49,13 +49,13 @@ using an example application.
 
 ### Serving with `http-server`
 
-Because `ng serve` does not work with service workers, you must use a separate HTTP server to test your project locally. You can use any HTTP server. The example below uses the [http-server](https://www.npmjs.com/package/http-server) package from npm. To reduce the possibility of conflicts, test on a dedicated port.
+Because `ng serve` does not work with service workers, you must use a separate HTTP server to test your project locally. You can use any HTTP server. The example below uses the [http-server](https://www.npmjs.com/package/http-server) package from npm. To reduce the possibility of conflicts, test on a dedicated port and disable caching.
```
Comment (gkalpak) [inline aio/content/guide/service-worker-getting-started.md]: As it is now, this sentence makes it sound as if disabling the cache reduces the possibility of conflicts (which is not true). I would change it to something like:

> To reduce the possibility of conflicts and avoid serving stale content, test on a dedicated port and disable caching.

**Review by gkalpak — CHANGES_REQUESTED** [review]
Good catch 👍

File: aio/content/guide/service-worker-getting-started.md (hunk 1 above)
Comment (LiadIdan) [reply aio/content/guide/service-worker-getting-started.md]: @gkalpak, done!

File: aio/content/guide/service-worker-getting-started.md [hunk 2]
```diff
@@ -49,13 +49,13 @@ using an example application.
...
 
-Because `ng serve` does not work with service workers, you must use a separate HTTP server to test your project locally. You can use any HTTP server. The example below uses the [http-server](https://www.npmjs.com/package/http-server) package from npm. To reduce the possibility of conflicts, test on a dedicated port.
+Because `ng serve` does not work with service workers, you must use a separate HTTP server to test your project locally. You can use any HTTP server. The example below uses the [http-server](https://www.npmjs.com/package/http-server) package from npm. To reduce the possibility of conflicts and avoid serving stale content, test on a dedicated port and disable caching.
 
 To serve with `http-server`, change to the directory containing your web files and start the web server: 
 
 ```sh
 cd dist/*project-name*
-http-server -p 8080
+http-server -p 8080 -c-1
```
Comment (gkalpak) [inline aio/content/guide/service-worker-getting-started.md]: Not directly related to this PR, but wouldn't it be better to save one step with:

```
http-server -p 8080 -c-1 dist/<project-name>
```

**Review by gkalpak — APPROVED** [review]
LGTM (+ one minor optional suggestion)

File: aio/content/guide/service-worker-getting-started.md (hunk 2 above)
Comment (LiadIdan) [reply aio/content/guide/service-worker-getting-started.md]: Yeah good point :smiley:

File: aio/content/guide/service-worker-getting-started.md [hunk 3]
```diff
@@ -122,7 +122,7 @@ next step is understanding how updates work.
 ```sh
 ng build --prod
 cd dist
-http-server -p 8080
+http-server -p 8080 -c-1 dist/<project-name>
```
Comment (gkalpak) [inline aio/content/guide/service-worker-getting-started.md]: Good catch. I had missed this one 😇

File: aio/content/guide/service-worker-getting-started.md [hunk 4]
```diff
@@ -49,13 +49,13 @@ using an example application.
 
 ### Serving with `http-server`
 
-Because `ng serve` does not work with service workers, you must use a separate HTTP server to test your project locally. You can use any HTTP server. The example below uses the [http-server](https://www.npmjs.com/package/http-server) package from npm. To reduce the possibility of conflicts, test on a dedicated port.
+Because `ng serve` does not work with service workers, you must use a separate HTTP server to test your project locally. You can use any HTTP server. The example below uses the [http-server](https://www.npmjs.com/package/http-server) package from npm. To reduce the possibility of conflicts and avoid serving stale content, test on a dedicated port and disable caching.
 
 To serve with `http-server`, change to the directory containing your web files and start the web server: 
 
 ```sh
 cd dist/*project-name*
```
Comment (gkalpak) [inline aio/content/guide/service-worker-getting-started.md]: You can now get rid of the `cd` commands (here and below) and update the corresponding text to not mention that you have to _"change to the directory containing your web app"_.

```diff
-To serve with `http-server`, change to the directory containing your web files and start the web server:
+To serve the directory containing your web files with `http-server`, run the following command:
```

**Review by gkalpak — CHANGES_REQUESTED** [review]
Almost there 😅

**Comment by LiadIdan** [comment]
@gkalpak so I need to change all the previous commit messages? if so maybe I'll open a new PR because I don't want to pull the repo to my machine :sweat_smile:

**Comment by LiadIdan** [comment]
Alright so I'm closing this one and will open a new PR in a bit :smile:

**Pull request closed without merging**